flang/lib/Optimizer/Builder/IntrinsicCall.cpp

```diff
@@ -400,6 +400,10 @@ static constexpr IntrinsicHandler handlers[]{
        {"values", asBox, handleDynamicOptional}}},
      /*isElemental=*/false},
     {"dble", &I::genConversion},
+    {"derfc_scaled",
```

Comment:
Once you handle these intrinsics as specific instances of the generic ones, you should not need any changes to lowering.